### PR TITLE
First import

### PR DESCRIPTION
**Pull request by jonathanpallant**
This PR is a WIP, as I import the completed decks from teaching-material2.

Note, currently suffers from https://github.com/ferrous-systems/material-templates/issues/14 because I based it off https://github.com/ferrous-systems/material-templates/releases/tag/v0.2.0

Needs that fixing first.

### CODE REVIEW

File: training-slides/src/basic-types.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# Basic types
+
+## ints
```
Comment (Mirabellensaft) [inline training-slides/src/basic-types.md:3]:
```suggestion
## Integers
```

File: training-slides/src/basic-types.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+```rust
+let x = 123_456;   // underscore as separator
+let x = 0x12;      // prefix 0x to indicate hex value
+let x = 0o23;      // prefix 0o to indicate octal value
+let x = 0b0001;    // prefix 0b to indicate binary value
+let x = b'a';      // A single u8
+```
+
+## Architecture-dependent numbers
```
Comment (Mirabellensaft) [inline training-slides/src/basic-types.md:23]:
```suggestion
## Architecture-dependent Numbers
```

**Review by Mirabellensaft — APPROVED** [review]
Once all the content is there, we need to tweak it more agressively into the shape of the curriculum.

**Comment by jonathanpallant** [comment]
> Once all the content is there, we need to tweak it more agressively into the shape of the curriculum.

Ah, I thought that work had been done already.

File: training-slides/src/glossary.md [hunk 3]
```diff
@@ -1 +1,29 @@
...
+|                               Training                                | Consists of different modules over a series of days or half-days                              |
+|                                Wash-up                                | Last 15 minutes of a training day or half-day for recap, open questions, outlook for next day |
+|                                Opening                                | First 15 minutes of a training day or half-day, with an ice-breaker and recaps, day's plan    |
+|                                Quizzes                                | Mini-tests of the training material                                                           |
+|                             Ice Breakers                              | Brief warm-up activities to get the training started, usually short Questions                 |
+| [Training Material](https://github.com/ferrous-systems/rust-training) | These training materials                                                                      |
+
+
+### Ice breakers
```
[the remark this answers is not in the document]
Comment (jonathanpallant) [reply training-slides/src/glossary.md:21]: Can you submit a PR to remove/move them?

[the remark this answers is not in the document]
Comment (jonathanpallant) [reply training-slides/src/glossary.md:21]: Actually I'll remove them from here. You can pull them from the history or the teaching-material2 repo.

File: training-slides/src/start_here.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Start Here
+
+Make sure to have `rustup`, `VSCode` and the `rust-analyzer` VSCode extension installed.
+
+The structure of the course is as follows:
```
[the remark this answers is not in the document]
Comment (jonathanpallant) [reply training-slides/src/start_here.md]: I've re-worded it.

**Comment by jonathanpallant** [comment]
@Mirabellensaft are you happy with the final edits? At least happy enough to see it committed - we can do more edits later.

**Comment by Mirabellensaft** [comment]
Yup, Happy enough to be committed.

**Pull request merged**